tests/features/mocking_async.py

```diff
@@ -0,0 +1,201 @@
+"""Tests for flexmock mocking feature."""
+# pylint: disable=missing-docstring,no-self-use,no-member,too-many-lines
+import unittest
+import pytest
+
+from flexmock import exceptions, flexmock
+from tests import some_module
+from tests.utils import assert_raises
+
+
+class MockingAsyncTestCase(unittest.IsolatedAsyncioTestCase):
```

Comment:
Looks like this is not yet supported in 3.7 but it has reached EOL so we should update the supported python versions